### ISSUE TITLE
Frontend refactoring

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#93]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Code is cleaned up
* Architecture is updated to the expected future needs
* Interfaces are updated and in documentation

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by veitmo** [comment]
@annikakrause: Hier mal ein Issue zu dem Refactoring und Testing. Falls dir weitere/detailliertere Acceptance Criteria einfallen, darfst du die auch jederzeit gerne ergänzen :)
